README.md

```diff
@@ -277,22 +277,8 @@ to "Block".
 Run `./hack/e2e.sh -h` to view help.
 
 ### View CI Results
-[GCE](https://k8s-testgrid.appspot.com/sig-storage#gce&include-filter-by-regex=PersistentVolumes-local)
-
-[GKE](https://k8s-testgrid.appspot.com/sig-storage#gke&include-filter-by-regex=PersistentVolumes-local)
-
-[GCE Slow](https://k8s-testgrid.appspot.com/sig-storage#gce-slow&include-filter-by-regex=PersistentVolumes-local)
-
-[GKE Slow](https://k8s-testgrid.appspot.com/sig-storage#gke-slow&include-filter-by-regex=PersistentVolumes-local)
-
-[GCE Serial](https://k8s-testgrid.appspot.com/sig-storage#gce-serial&include-filter-by-regex=PersistentVolumes-local)
-
-[GKE Serial](https://k8s-testgrid.appspot.com/sig-storage#gke-serial&include-filter-by-regex=PersistentVolumes-local)
-
-[GCE Alpha](https://k8s-testgrid.appspot.com/sig-storage#gce-alpha&include-filter-by-regex=PersistentVolumes-local)
-
-[GKE Alpha](https://k8s-testgrid.appspot.com/sig-storage#gke-alpha&include-filter-by-regex=PersistentVolumes-local)
 
+Check testgrid [sig-storage-local-static-provisioner](https://testgrid.k8s.io/sig-storage-local-static-provisioner) dashboard.
```

Comment:
I noticed testgrid results are not listing the test cases.  I think you need to pass through the ARTIFACTS env variable to get it to work:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/184

Reply:
Also is this path going to be correct after https://github.com/kubernetes/test-infra/pull/10696?

Reply:
Path will be correct.

I'll take a look at ARTIFACTS.